config/smfcfg.yaml

```diff
@@ -21,6 +21,7 @@ configuration:
     brokerUri: "sd-core-kafka-headless"
     brokerPort: 9092
     topicName: "sdcore-data-source-smf"
+    disableKafka: false
```

Comment:
I'm not sure about the double negation, why not have enableKafka (and default to True) instead of disableKafka (and default to False)

Reply:
The reason I did it that way was to keep the code simple and maintain backwards compatibility. When a boolean field is not specified in the configuration file, it will use the default value of the type, which is `false` for a `bool`.

I am currently working on reversing the logic, but that will also mean changing the type to a `*bool` to be able to differentiate between `false` and no value provided. The code doing the unmarshaling will also change to detect a `nil` and insert `true` instead.

Reply:
The last commit contains the change, please let me know what you think.